docs/platform/release-notes/tizen-4-0-m2.md

```diff
@@ -1,9 +1,10 @@
 # Tizen 4.0 Public M2
 
-Release Date: 01 Nov, 2017
+Release date: Nov. 11, 2017
```

Comment:
Question. Nov. 01 is changed to Nov. 11.
But it looks like Nov. 01 is correct. 
Would you please check?

Reply:
Fixed.